ulib/FStar.UInt.Base.fsti

```diff
@@ -0,0 +1,106 @@
+module FStar.UInt.Base
+open FStar.UInt.Types
+open FStar.Mul
+
+#reset-options "--initial_fuel 0 --max_fuel 0"
```

Comment:
Can you remove this `#reset-options`? Setting options in an fsti has somewhat confusing semantics due to interleaving and we shouldn't need it in this file.